system-int-test/test/cmr/system_int_test/search/all_granule_search_test.clj

```diff
@@ -77,26 +77,28 @@
         result14 (search/find-refs :granule params8)
         result15 (search/find-refs :granule params9)
         result16 (search/find-refs :granule params10)
-        err-msg "The CMR does not allow querying across granules in all collections. To help optimize your search, you should limit your query using conditions that identify one or more collections, such as provider, provider_id, concept_id, collection_concept_id, short_name, version or entry_title. For any questions please contact cmr-support@earthdata.nasa.gov."
+        err-msg (seq '("The CMR does not allow querying across granules in all collections. To help optimize your search, you should limit your query using conditions that identify one or more collections, such as provider, provider_id, concept_id, collection_concept_id, short_name, version or entry_title. For any questions please contact cmr-support@earthdata.nasa.gov."))
```

Comment:
You shouldn't have to make this kind of changes in this namespace.

Reply:
I'm at a loss for words as to why this test was failing to begin with, because none of my changes should've affected how error message was returned, but now running it without my change it's no longer failing locally like it was before. Undid the change to this file.